fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -349,6 +349,7 @@ terminal String
     KW_ENTER,
     KW_ERRORS,
     KW_EVENTS,
+    KW_EVERY,
```

Comment:
maybe we can reuse keyword `INTERVAL`? 
And in MySQL, there are many stmt like `INTERVAL 1 DAY`, and I found in PG:
https://www.postgresql.org/docs/current/ddl-partitioning.html

we can refer as:
```
PARTITION BY RANGE(date)
FROM ("2022-10-11") TO ("2022-11-10") INTERVAL 1 DAY/MONTH/WEEK/YEAR

PARTITION BY RANGE(int_col)
FROM ("10") TO ("100") INTERVAL 10
```

Reply:
And I see the keyword `KW_MULTI`, but i didn't see examples.
Could you please given more example about the new syntax.

Reply:
Reusing keywords is a good idea, I'll give it a try